### PR TITLE
Project description

### PR DESCRIPTION
**Pull request by alensiljak**
A minor update to clarify the goals, history, and the purpose of the project/app.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -5,19 +5,24 @@
 
 * Orgzly Revived
 
-Orgzly is an outliner for taking notes and managing to-do lists.
+Orgzly Revived is a community-maintained version of Orgzly, as the development of the original app is no longer active.
 
-You can keep notebooks stored in plain-text and have them synchronized
-with a directory on your mobile device, SD card, WebDAV server or Dropbox.
+Orgzly Revived project aims to continue development of the Orgzly application, adding useful features,
```
Comment (cbkerr) [inline README.org:10]:
```suggestion
Orgzly Revived aims to continue development of the Orgzly application, adding useful features, 
```

File: README.org [hunk 2]
```diff
@@ -5,19 +5,24 @@
...
 * Orgzly Revived
 
-Orgzly is an outliner for taking notes and managing to-do lists.
+Orgzly Revived is a community-maintained version of Orgzly, as the development of the original app is no longer active.
 
-You can keep notebooks stored in plain-text and have them synchronized
-with a directory on your mobile device, SD card, WebDAV server or Dropbox.
+Orgzly Revived project aims to continue development of the Orgzly application, adding useful features, 
+fixing bugs, keeping up-to-date with technological evolution, and generally maintaining the 
+[Org Mode](https://orgmode.org/) support on Android.
```
Comment (cbkerr) [inline README.org:12]: Use org mode link format

```suggestion
[[https://orgmode.org/)][Org Mode]] support on Android.
```

File: README.org [hunk 3]
```diff
@@ -5,19 +5,24 @@
...
+
+[Orgzly](https://orgzly.com/) is an outliner for taking notes and managing to-do lists. The source code
```
Comment (cbkerr) [inline README.org:14]:
```suggestion
[[https://orgzly.com/][Orgzly]] is an outliner for taking notes and managing to-do lists. The source code
```

File: README.org [hunk 4]
```diff
@@ -5,19 +5,24 @@
...
+[repository](https://github.com/orgzly/orgzly-android) is available on GitHub but is no longer actively
```
Comment (cbkerr) [inline README.org:15]:
```suggestion
[[https://github.com/orgzly/orgzly-android][repository]] is available on GitHub but is no longer actively 
```

File: README.org [hunk 5]
```diff
@@ -5,19 +5,24 @@
...
 
-You can keep notebooks stored in plain-text and have them synchronized
-with a directory on your mobile device, SD card, WebDAV server or Dropbox.
+Orgzly Revived project aims to continue development of the Orgzly application, adding useful features, 
+fixing bugs, keeping up-to-date with technological evolution, and generally maintaining the 
+[Org Mode](https://orgmode.org/) support on Android.
+
+[Orgzly](https://orgzly.com/) is an outliner for taking notes and managing to-do lists. The source code
+[repository](https://github.com/orgzly/orgzly-android) is available on GitHub but is no longer actively 
+maintained. We have used this code as the base for Orgzly Revived.
```
Comment (cbkerr) [inline README.org:16]: I think we should add a note about getting involved and transferring old issues and PRs as you brought up in your discussion post! https://github.com/orgzly-revived/orgzly-android-revived/discussions/2

```suggestion
maintained. We have used this code as the base for Orgzly Revived. Please feel free to get involved in the project on GitHub by contributing issues, ideas, or features! We generally plan to leave existing open issues in the original repository and reference them here as we address them.
```

**Review by cbkerr — APPROVED** [review]
I like it! I think we should add a call to contribute and suggestion on what to do with existing issues like you brought up earlier. See if you like my suggestion

File: README.org (hunk 3 above)
Comment (alensiljak) [reply README.org:14]: Good catch!

File: README.org (hunk 5 above)
Comment (alensiljak) [reply README.org:16]: We should, at some point, add a Contributing doc. For now, I've created a separate section.
#45

File: README.org (hunk 1 above)
Comment (alensiljak) [reply README.org]: See the reworded version. Here I actually did specifically mean the project, the Github organization, if you will. That is the "community" behind, as opposed to the app itself. Even though it is currently pretty-much the same thing.

**Review by amberin — APPROVED** [review]
Very good!

File: README.org [hunk 6]
```diff
@@ -5,24 +5,30 @@
 
 * Orgzly Revived
 
-Orgzly is an outliner for taking notes and managing to-do lists.
+Orgzly Revived is a community-maintained version of Orgzly, as the development of the original app is no longer active.
 
-You can keep notebooks stored in plain-text and have them synchronized
-with a directory on your mobile device, SD card, WebDAV server or Dropbox.
+Orgzly Revived project aims to continue development of the Orgzly application. The rebranding is due
```
Comment (cbkerr) [inline README.org:10]:
```suggestion
The Orgzly Revived project aims to continue development of the Orgzly application. The rebranding is due 
```

**Review by cbkerr — APPROVED** [review]
Looks like adding a word dismissed @amberin's review - didn't mean for that.
Looks good!

**Review by amberin — APPROVED** [review]

**Pull request merged**